app.rb

```diff
@@ -0,0 +1,158 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  MENU_OPTIONS = {
+    '1' => :list_books,
+    '2' => :list_people,
+    '3' => :create_person,
+    '4' => :create_book,
+    '5' => :create_rental,
+    '6' => :list_all_rentals,
+    '7' => :exit_program
+  }.freeze
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def start
+    loop { choose_option }
+  end
```

Comment:
- [ ] @fatmahussein  Great work adding the UI, however list of options should be in the main.rb file, this is a project requirement as shown below
![image](https://github.com/fatmahussein/school-library/assets/61408860/52c24ed0-768e-4bca-907a-a8f73af2c2a1)